Sources/SwiftNavigation/UIBinding.swift

```diff
@@ -677,13 +676,12 @@ where Base.Value: Hashable {
   }
 }
 
-private final class _UIBindingEnumToOptionalCase<Base: _UIBinding, Case>: _UIBinding, @unchecked
-  Sendable
+private final class _UIBindingEnumToOptionalCase<Base: _UIBinding, Case>: _UIBinding
```

Comment:
This lose the `Sendable`? Or is it implicit and can we remove the explicit one above?

```suggestion
private final class _UIBindingEnumToOptionalCase<Base: _UIBinding, Case>: _UIBinding, Sendable
```

Reply:
`_UIBinding` is `: Sendable` so we already get it.